src/Text/index.js

```diff
@@ -1,79 +1,67 @@
 // @flow
 import * as React from "react";
-import * as tokens from "@kiwicom/orbit-design-tokens";
-
-const colorText = {
-  primary: tokens.colorTextPrimary,
-  secondary: tokens.colorTextSecondary,
-  attention: tokens.colorTextAttention,
-};
-
-const weightText = {
-  regular: tokens.fontWeightNormal,
-  bold: tokens.fontWeightBold,
-};
-
-const sizeText = {
-  large: tokens.fontSizeTextLarge,
-  normal: tokens.fontSizeTextNormal,
-  small: tokens.fontSizeTextSmall,
-};
+import defaultTokens from "@kiwicom/orbit-design-tokens";
+import styled from "styled-components";
+import { withTheme } from "theming";
+import createComponentFromTagProp from "react-create-component-from-tag-prop";
 
 type Props = {
-  type: $Keys<typeof colorText>,
-  size: $Keys<typeof sizeText>,
-  weight: $Keys<typeof weightText>,
+  type: "primary" | "secondary" | "attention",
+  size: "small" | "normal" | "large",
+  weight: "regular" | "bold",
```

Comment:
Why is here defined `regular` and token from orbit-design-tokens is called `fontWeightNormal` ?

Reply:
Fixed.